application/view-admin/omeka/site-admin/index/theme-selector.phtml

```diff
@@ -29,5 +35,8 @@
         <input type="hidden" name="o:theme" value="<?php echo $theme->getId(); ?>" <?php echo ($defaultActive) ? '' : 'disabled'; ?>>
     </div>
     <?php endif; ?>
-    <?php endforeach; ?>
+    <?php
+    endif;
+    endforeach;
+    ?>
```

Comment:
We may want  to display errors here as well...

This goes to part of the weirdness of the theme selector: it's the only place where these themes are viewable, so if we want to show an admin what is wrong so they can fix it or ask the forums, we need to do it here. Otherwise, the failure is silent.